tests/multi_ptr/multi_ptr_local_accessor_constructor.h

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for multi_ptr local_accessor constructors
+//
+*******************************************************************************/
+#ifndef __SYCLCTS_TESTS_MULTI_PTR_LOCAL_ACCESSOR_CONSTRUCTORS_H
+#define __SYCLCTS_TESTS_MULTI_PTR_LOCAL_ACCESSOR_CONSTRUCTORS_H
+
+#include "../common/common.h"
+#include "../common/section_name_builder.h"
+#include "multi_ptr_common.h"
+
+namespace multi_ptr_local_accessor_constructors {
+
+template <typename T, typename Address_SpaceT, typename DimensionT>
```

Comment:
```suggestion
template <typename T, typename AddrSpaceT, typename DimensionT>
```
Just a suggestion. Feel free to ignore)

Reply:
Done